gateways/istio-egress/templates/autoscale.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.autoscaleMin }}
+{{ $gateway := index .Values "gateways" "istio-egressgateway" }}
```

Comment:
We're no longer using this - each egress is 'a-la-carte', installed independently

Reply:
I though the whole point of the prefix was for backwards compatibility with 1.1, which is configured via `gateways.istio-egressgateway`

Reply:
Backward compat where it makes sense and doesn't conflict with the new design. 

For security and operation reasons we want to isolate the components in different namespaces - so secrets, etc can be managed by different teams and deployed/operated independently.

Reply:
they are still separated the only difference is how they are configured through helm
--set gateways.egress.something vs --set egress.something.  the 2nd is probably better ux but 1st isn't so bad and is compatible.

Reply:
Discussed offline, we will see if we can have both options work for now